db/migrate/20161112155558_add_attachment_image_to_admin_testimonials.rb

```diff
@@ -1,11 +0,0 @@
-class AddAttachmentImageToAdminTestimonials < ActiveRecord::Migration
```

Comment:
i did not touch all those at all but how ???? they are all in master right , why merging master does not update the master